userdatamodel/user.py

```diff
@@ -44,7 +45,7 @@ def __setitem__(self, key, value, _sa_initiator=None):
 
 
 class User(Base):
-    __tablename__ = "User"
+    __tablename__ = 'User'
```

Comment:
is there any chance we can lowercase this or is it conflicting with something else?